medium/__init__.py

```diff
@@ -7,84 +7,139 @@
+    an authorization code from the user. Send them to the URL given by
+    ``Client.get_authorization_url()`` and then have them enter their
+    authorization code to exchange it for an access token.
+
+    >>> client.access_token
+    None
+    >>> redirect_url = "https://yoursite.com/callback/medium"
+    >>> authorize_url = client.get_authorization_url(
+    ...     state="secretstate",
+    ...     redirect_url=redirect_url,
+    ...     scopes=["basicProfile", "publishPost"]
+    ... )
+    >>> print 'Go to: {}'.format(authorize_url)
+    Go to: https://medium.com/m/oauth/authorize?scope=basicProfile%2Cpublish...
+    >>> print 'Copy the authorization code.'
+    Copy the authorization code.
+    >>> authorization_code = raw_input('Enter the authorization code here: ')
+    >>> client.exchange_authorization_code(authorization_code, redirect_url)
+    >>> client.access_token
+    ...
+
+    The access token will expire after some time. To refresh it:
+
+    >>> client.exchange_refresh_token()
+
+    Once you have an access token, you can use the rest of the client's
+    methods. For example, to get the profile details of user identified by the
+    access token:
+
+    >>> user = client.get_current_user()
+
+    And to create a draft post:
+
+    >>> post = client.create_post(
+    ...     user_id=user["id"],
+    ...     title="Title",
+    ...     content="<h2>Title</h2><p>Content</p>",
+    ...     content_format="html", publish_status="draft"
+    ... )
+
+    """
     def __init__(self, application_id=None, application_secret=None,
-                 access_token=None):
+                 access_token=None, refresh_token=None):
         self.application_id = application_id
         self.application_secret = application_secret
         self.access_token = access_token
+        self.refresh_token = refresh_token
 
     def get_authorization_url(self, state, redirect_url, scopes):
         """Get a URL for users to authorize the application.
 
-        :param str state: A string that will be passed back to the redirect_url
-        :param str redirect_url: The URL to redirect after authorization
-        :param list scopes: The scopes to grant the application
+        :param str state:
+            A string that will be passed back to the redirect_url
+        :param str redirect_url:
+            The URL to redirect after authorization
+        :param list scopes:
+            The scopes to grant the application
         :returns: str
         """
-        qs = {
+        return "https://medium.com/m/oauth/authorize?" + urlencode({
             "client_id": self.application_id,
-            "scope": ",".join(scopes),
-            "state": state,
             "response_type": "code",
             "redirect_uri": redirect_url,
-        }
+            "scope": ",".join(scopes),
+            "state": state,
+        })
 
-        return "https://medium.com/m/oauth/authorize?" + urlencode(qs)
+    def _get_tokens(self, **kwargs):
```

Comment:
Since `_get_tokens` only ever passes `form_data`, maybe you should have it just accept that as a named argument? I feel like accepting and just passing on kwargs can make things harder to follow